app/controllers/mfa_confirmation_controller.rb

```diff
@@ -20,7 +20,7 @@ def password
   end
 
   def handle_valid_password
-    if current_user.totp_enabled?
```

Comment:
`totp_enabled?` was defined by the gem but always returned false because the method checked if `otp_secret_key` was present, and we currently ignore that column on the user